.github/workflows/ci-code.yml

```diff
@@ -64,8 +64,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
 
     - name: Setup environment
-      # Note: The virtual environment in .venv was created by uv in previous step
-      run: source .venv/bin/activate && .github/workflows/setup.sh
```

Comment:
The updated setup-uv action now activates the virtual environment automatically.